### ISSUE TITLE
Unable to create a User

### ISSUE THREAD

**Issue by wranglatang** [linuxserver/docker-clarkson#11]
Hello,

I have deployed this container on my UNRAID server and connected it to a 5.7 version of MySQL, created the DB scheme before running the container. However I am unable to create an account. Looks like a similar problem is being seen by others. I have tried to comment on the post but got no response. [GitHub Issue Here](https://github.com/linuxserver/Clarkson/issues/17)

Anything you can help with?

**Comment by wranglatang** [comment]
Figured it out. It seems that I had some problems with mysql - probably through all the development work I cased something to break somewhere. So I deleted the Clarkson database and user then ran this command on the mysql docker 

`mysql_upgrade -uroot -p --force`

[Link to Detail](https://www.myguysolutions.com/2013/10/30/how-to-resolve-mysql-error-code-1548-cannot-load-from-mysql-proc-the-table-is-probably-corrupted/)

After that - It worked!
